app.py

```diff
@@ -356,12 +378,12 @@ def get_active_traefik_svcs() -> Dict[str, time.time]:
                     else:
                         logger.debug({"message": "Skipping because {} not in {}".format(cfg['narr_img'], image_name)})
                 else:
-                    logger.debug({"message": "Skipped {} because it didn't match prefix {}".format(name, cfg['container_prefix'])})
+                    logger.debug({"message": "Skipped {} because it didn't match prefix {}".format(name, prefix)})
             return(narr_activity)
         else:
             raise(Exception("Error querying {}:{} {}".format(cfg['traefik_metrics'], r.status_code, r.text)))
     except Exception as e:
-        exc_type, exc_obj, tb = sys.exc_info()
+        _, _, tb = sys.exc_info()
```

Comment:
This is the first time I've encountered having to do this to fix the codacy alert. Very cool!